### ISSUE TITLE
Minlevel not properly respected in IE?

### ISSUE THREAD

**Issue by tomturner123** [openseadragon/openseadragon#1389]
Hi. I love openseadragon and have created loads of images. As the lower levels are so small I only included levels 8 and above but this works great using the minLevel option to prevent them being loaded. However on internet explorer 11 (and possibly others?) a warning is generated that tile /1/0_0.jpg doesnt exist. Everything works properly from a user pov but the apache logs fill up really fast. Is there any way to prevent this? thank you in advance.

**Comment by iangilman** [comment]
Strange! The minLevel is working on other browsers though?

**Comment by tomturner123** [comment]
Yes, but the minlevel works on ie - it just generates a warning. No warnings generated from the browsers on my mac as far as I can see

________________________________

**Comment by iangilman** [comment]
Sounds like there are server calls to images it shouldn't be trying to get. What does it look like in the network tab of your inspector? Is it trying to load the rest of the pyramid or just that one stray tile?

It's odd that it's only happening in IE. I wonder if there's some other difference, like different screen size or pixel density.

What sort of image is it? DZI? How are you setting the minLevel?

**Comment by tomturner123** [comment]
I think you’re probably right, and my reference to IE might be incorrect. It doesn’t happen every time so it might just be that most users are using IE. I think I may have since spotted a couple of chrome references. I can reproduce the error on a windows VM running on my laptop, but the same VM running on my imac doesn’t generate the error. No idea what the difference is. 

 

 

All the images are dzi. I just add minLevel to the viewer.open call e.g:

 

viewer.open({

    image/tileSource: "XXX",

    minLevel: 8

});

**Comment by iangilman** [comment]
Do both your laptop and iMac have retina screens? That might make a difference.

Anyway, I don't believe you can add a minLevel into the `open` call like that... It also doesn't look like you can include in the DZI file. One option would be to create a `DziTileSource` yourself and pass that into `open`. Another thing you might try is opening it as you are now, but then doing: 

```
viewer.addHandler('open', function() {
  viewer.world.getItemAt(0).source.minLevel = 8;
});
```

**Comment by iangilman** [comment]
Incidentally, OSD normally doesn't load the smallest tiles, instead starting with the biggest tile that contains the entire image. Perhaps on a number of computers this is level 8 for your images...
